docs/pricing/free.mdx

```diff
@@ -0,0 +1,40 @@
+| ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Description**              | - AI editor assistant for hobbyists or light usage                                                                                                       | - AI and search for growing organizations hosted on our cloud                                                                                             |
+| **Price**                    | - $0/month <br /> - 1 user                                                                                                                               | - $19/user/month <br /> - Up to 50 devs                                                                                                                   |
+|**AI features**              | - Unlimited autocomplete <br /> - 200 chat messages and prompts per month <br /> - Multiple LLM choices for chat <br /> - Connect to local Ollama models | - Unlimited autocomplete and chat <br /> - More powerful LLMs for chat (GPT-4o, Claude 3 Opus) <br /> - Intent detection and integrated search results    |
+| **Code Search features** | N/A                                                                                                                                                      | - Code Search <br /> - Symbol Search                                                                                                                      |
+| **Deployment types**     | - Multi-tenant Coud                                                                                                                                      | - Multi-tenant Cloud <br /> - Private Workspace <br /> - Privately indexed code (100 repos)                                                               |
+| **Compatibility**        | - VS Code, JetBrains IDEs, Visual Studio, and Eclipse <br /> - All popular coding languages <br /> Natural language search <br /> - All major code hosts | - VS Code, JetBrains IDEs, Visual Studio, and Eclipse <br /> - All popular coding languages <br /> Natural language search <br /> - Code hosted on GitHub |
+| **Support**              | - Community support only                                                                                                                                 | - Community support only                                                                                                                                  |
+
+## Upgrading to Enterprise Starter
```

Comment:
We don't want to use upgrade language since today it won't replace your pro subscription.